js/src/scrollspy.js

```diff
@@ -221,30 +221,35 @@ class ScrollSpy extends BaseComponent {
 
     const link = SelectorEngine.findOne(queries.join(','), this._config.target)
 
-    link.classList.add(CLASS_NAME_ACTIVE)
-    if (link.classList.contains(CLASS_NAME_DROPDOWN_ITEM)) {
-      SelectorEngine.findOne(SELECTOR_DROPDOWN_TOGGLE, link.closest(SELECTOR_DROPDOWN))
-        .classList.add(CLASS_NAME_ACTIVE)
+    if (link !== undefined && link !== null) {
```

Comment:
```js
    if (link) {
```

should be enough, no? Just make sure you swap the conditions if you go with this approach.

Reply:
Thank you for the comment. That's better indeed. Will make the change asap

Reply:
Or actually, just do `if (!link)`. But this needs tests for sure either way.